nvi-evaluator/src/main/java/no/sikt/nva/nvi/evaluator/calculator/NviCandidate.java

```diff
@@ -1,7 +1,8 @@
 package no.sikt.nva.nvi.evaluator.calculator;
 
-import no.sikt.nva.nvi.evaluator.model.CandidateResponse;
+import java.net.URI;
+import java.util.List;
 
-public record NviCandidate(CandidateResponse response) implements CandidateType {
+public record NviCandidate(List<URI> approvalAffiliations) implements CandidateType {
```

Comment:
I find the record name "NviCandidate" somewhat confusing when it only contains a list of affiliations uris, and no data about the candidate. Rename to something more appropriate?